.golangci.yml

```diff
@@ -3,14 +3,6 @@ run:
 #   # timeout for analysis, e.g. 30s, 5m, default is 1m
 #   timeout: 5m
 
-linters:
-  disable-all: true
-  enable:
-    - errcheck
-    - gofmt
-    # - goimports
-    # - golint
-    - maligned
```

Comment:
I'd significantly prefer the approach of disabling all, and manually re-enabling, but doing so to explicitly add what we want to add

Reply:
ah, that way there's a list and we know which ones!

ok, can do :)

Reply:
it is kinda hilarious, the degree to which you made a truly excellent call here.